### DISCUSSION TITLE
Icon request: icon-file-lightbulb

### DISCUSSION THREAD

**Discussion by benjino** [FortAwesome/Font-Awesome#21219] [Icon Requests]
### What can it be used for?

Let's ask what does it mean first. "Evidence, typically derived from an experiment or pilot project, which demonstrates that a design concept, business proposal, etc., is feasible." It can be used to represent your business's services, your area of expertise, or that you have an idea and documentation that supports proof of concept, hence the use of the lightbulb for concept (the light goes on) and the file (documentation, report or study). The first image use example has a border around what is being proposed; the second image use example is what is being proposed.

### Example image (optional)

![proof_of_concept_1](https://user-images.githubusercontent.com/3208630/43478791-ae02342a-94b3-11e8-8808-f48467e9da0a.jpg)
![proof_of_concept_2](https://user-images.githubusercontent.com/3208630/43478792-ae20e406-94b3-11e8-9b5c-af9364fcd77d.jpg)

### Icon request checklist

- [ X ] This is a single icon.
- [ X ] The title starts with "Icon request: Yes.
- [ X ] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate. Yes, I have.
- [ X ] I have included at least one example image if it could be helpful (optional). Yes, two examples.
- [ X ] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺). It's an icon based on a well known concept or idea, in particular a phrase that is often used in science or business, "Proof of Concept." I created this icon by simply combining FA 5 Pro lightbulb and file icons, and am using it in a website I designed and maintain and update for a client of mine.

**Comment by tagliala** [comment]
Hi,

you can implement this icon by using the masking and stacking features of  the SVG framework

http://jsfiddle.net/tagliala/9tercwpy/28/

Ref:
- https://fontawesome.com/how-to-use/on-the-web/styling/masking
- https://fontawesome.com/how-to-use/on-the-web/styling/layering

**Comment by benjino** [comment]
Thank you for sharing Tagliala.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
